practiceapp/src/main/java/com/practiceapp/practiceapp/entity/PostEntity.java

```diff
@@ -23,7 +23,7 @@
 
     @Id
     @Field
-    private String post_id;
+    private String _id;
```

Comment:
Just to prevent inconsistency it may be better to keep the field name as post_id.

Reply:
Thanks for informing. I committed the change, can you please take a look?

Reply:
Perfect, thank you.